include/picongpu/particles/flylite/IFlyLite.hpp

```diff
@@ -77,7 +77,7 @@ namespace flylite
              */
             PMACC_STATIC_ASSERT_MSG(
                 false && sizeof(T_IonSpecies) != 0,
-                FLYlite_the_update_method_for_ion_population_kinetics_is_not_implemented
+                FLYlite_the_update_method_for_ion_population_kinetics_is_not_implemented,
```

Comment:
Why are these commas after the last argument for a macro required?

Reply:
Apparently the standard requires the comma before the__VA_ARGS__ to be always there, even if the variadic arguments are empty. C++14 final draft 16.3.4:
```
If the identifier-list in the macro definition does not end with an ellipsis, the number of arguments (including
those arguments consisting of no preprocessing tokens) in an invocation of a function-like macro shall equal
the number of parameters in the macro definition. Otherwise, there shall be more arguments in the invocation
than there are parameters in the macro definition (excluding the ...)
```
It strictly says "more", not allowing for equal.

Reply:
Thx for explaining.